From b49dbaa0391f05adefc6384ea61cb9e60c4a452a Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Tue, 2 Mar 2021 18:58:04 +0100
Subject: [PATCH 112/191] mm: slub: Don't enable partial CPU caches on
 PREEMPT_RT by default

SLUB's partial CPU caches lead to higher latencies in a hackbench
benchmark.

Don't enable partial CPU caches by default on PREEMPT_RT.

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
---
 init/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/Kconfig b/init/Kconfig
index 37686a22a769..45f2231c7131 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -1974,7 +1974,7 @@ config SHUFFLE_PAGE_ALLOCATOR
 	  Say Y if unsure.
 
 config SLUB_CPU_PARTIAL
-	default y
+	default y if !PREEMPT_RT
 	depends on SLUB && SMP
 	bool "SLUB per cpu partial cache"
 	help
-- 
2.19.1

